Model/Config.php

```diff
@@ -665,4 +677,16 @@ public function getUpdateSqlLimit(): int
     {
         return self::UPDATE_SQL_LIMIT;
     }
+
+    /**
+     * @param int|null $scopeId
+     * @param string $entity
+     * @return boolean
+     * @throws LocalizedException
+     * @throws NoSuchEntityException
+     */
+    public function syncResetInProgress($scopeId, $entity)
```

Comment:
What about `isSyncResetInProgress`?

Reply:
Yes, we will make this change and update the PR